ProfileManager.js

```diff
@@ -543,41 +436,39 @@ export default class ProfileManager {
         authorizedDate: (new Date()).toISOString()
       }
     };
-    const {edv: usersEdv, invocationSigner} = await this.getProfileEdv({
-      profileId,
+
+    const {edvClient, invocationSigner} = await this.getUsersEdv({
```

Comment:
Note: Not for this PR, but we'll want to support passing in an edvClient+zcap+invocationSigner (maybe we just want an EdvDocument interface then?) for accessing the user's EDV as an alternative (need this for apps where the "users" EDV is controlled by an entirely different profile). That may imply a desire to refactor this API to just take an `EdvDocument` and have another call that creates an EdvDocument instance for writing a user.